src/styles/typeface/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Typeface
+description: If your service is on the service.gov.uk subdomain you must use the GDS Transport font
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+order: 5
+# headings:
+#   - text: Section break
+#     aliases: horizontal rule, hr
+
+#   - text: Links
+#     ignoreInPageNav: true
+#     ignoreInSearch: true
+
+#   - text: Lists
+#     ignoreInPageNav: true
+#     ignoreInSearch: true
+---
+
+{% from "_example.njk" import example %}
```

Comment:
We don't need to import `example` here since we don't have any examples on this page